client/state/analytics/actions.js

```diff
@@ -85,7 +85,7 @@ export const loadTrackingTool = trackingTool => ( {
 		analytics: [
 			{
 				type: ANALYTICS_TRACKING_ON,
-				payload: trackingTool,
+				payload: { trackingTool },
```

Comment:
Ah okay you also made updates here. Let's keep this small for now, and follow up We'd need to retest the privacy update settings otherwise